compose.yaml

```diff
@@ -69,7 +70,11 @@ services:
     image: smartonfhir/ctakes-covid:1.1.1
     environment:
       - ctakes_umlsuser=umls_api_key
-      - ctakes_umlspw=$UMLS_API_KEY
+      - ctakes_umlspw=${UMLS_API_KEY:-}
+    healthcheck:
+      test: echo 'Must set the UMLS_API_KEY environment variable.' && test -n "${UMLS_API_KEY:-}"
```

Comment:
This message only shows up if you manually go and do a `docker inspect` :shrug: But it's something.